Form/Type/DateRangeType.php

```diff
@@ -20,15 +20,28 @@
 class DateRangeType extends AbstractType
 {
     /**
-     * @var TranslatorInterface
+     * @var TranslatorInterface|null
+     *
+     * @deprecated translator property is deprecated since version 3.x, to be removed in 4.0
```

Comment:
We should use a common version notation, which is AFAIK `3.x`, not the concrete version.

I closed your PR, because there was no further action and I wanted to merge this stuff.